app/src/main/res/layout/fragment_home.xml

```diff
@@ -1,24 +1,73 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".ui.home.HomeFragment"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:id="@+id/homefragment">
 
 
-    <Button
-        android:id="@+id/btn_sign_out"
+    <fragment
+        class="com.google.android.gms.maps.SupportMapFragment"
+        android:id="@+id/map_fragment"
         android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        />
+
+    <ImageButton
+        android:id="@+id/btn_menu"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        android:src="@drawable/ic_baseline_menu_24"
+        android:background="@drawable/round_corner"
+        android:layout_marginTop="32dp"
+        android:layout_marginStart="16dp"
+        android:padding="10dp"
+        />
+
+    <ImageView
+        android:layout_width="wrap_content"
+        android:layout_height="40dp"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:layout_marginBottom="40dp"
+        android:src="@drawable/pin_icon"
+        />
+
+    <ImageButton
+        android:id="@+id/btn_current_location"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="30dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:src="@drawable/ic_baseline_my_location_24"
+        android:background="@drawable/round_corner"
+        android:layout_marginBottom="32dp"
+        android:layout_marginEnd="16dp"
         android:padding="10dp"
-        android:text="@string/btn_sign_outd"
-        android:textAllCaps="false"
-        android:textStyle="normal"
-        android:textSize="18dp"
-        android:textColor="@color/white"
-        style="@style/btnPrimaryRounded"
-        android:backgroundTint="@color/color_primary" />
+        />
+
+<!--    <Button-->
```

Comment:
as the sign out is going to be implement in another view i think you should delete the button instead of commenting it